app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :posts
```

Comment:
- Let's create many to one relationship here
`belongs_to :post`

Reply:
- [ ] You are almost there! However, what the previous reviewer meant is to be aware of the singular and plural, Ruby on Rails has many name conventions therefore it's different to say `user` and `users`. For models usually you use singular, so it would be better to say `belongs_to :user` or `:post`.